content/en/security_platform/cloud_workload_security/workload_security_rules.md

```diff
@@ -15,23 +15,27 @@ further_reading:
 
 ## Overview
 
-With Cloud Workload Security (CWS) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of rules to detect suspicious behavior. Before learning how to create custom rules below, it is recommended to learn the basics of Cloud Workload Security rules and the concept of Agent expressions. Refer to the [Agent expression docs][1] for more information.
+With Cloud Workload Security (CWS) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of rules to detect suspicious behavior. 
 
-The full set of rules for the Agent is called a policy. Datadog provides several [out-of-the-box Cloud Workload Security rules][2] for your convenience in the default policy.
+When you upgrade the Datadog Agent, you receive [bundled CWS Agent rules][1]. You can also write your own custom Agent rules. This guide covers Agent rules and how to create them.
 
-At some point, you will likely want to write your own custom rules for the Agent to use. Below are example rules, guidelines for optimization, and the underlying mechanics of rule efficiency.
+### Agent rules
 
-## Rule expressions
+An Agent Rule contains [Agent Expressions][2] that determine which activities the Agent collects. These captured events are then evaluated based on patterns of events described in a [rule][3]. 
 
-Rule expressions define behavior based on activity in your hosts and containers. For example, if you want to detect the following behavior, “the passwd command executed”, there are a few attributes to note.
+A full set of rules for the Agent is called a policy. Datadog provides you with several [out-of-the-box CWS Agent rules][1] in the default policy.
```

Comment:
A full set of rules for the Agent is called a policy. Datadog provides you with several [out-of-the-box CWS Agent](https://docs-staging.datadoghq.com/sarina/cws-rule-split/security_platform/default_rules/#cat-workload-security) Rules powered by the default Agent policy